ambry-cloud/src/main/java/com/github/ambry/cloud/azure/CosmosContainerDeletionEntry.java

```diff
@@ -25,6 +26,7 @@
 /**
  * Class representing container deletion status in cloud.
  */
+@JsonIgnoreProperties(ignoreUnknown = true)
```

Comment:
I think we should be worried if we find unknown fields. Lets throw exception in those cases instead of ignoring them.